osu.Game/osu.Game.csproj

```diff
@@ -323,6 +323,7 @@
     <Compile Include="Screens\Select\Options\BeatmapOptionsEditButton.cs" />
     <Compile Include="Screens\Select\Options\BeatmapOptionsOverlay.cs" />
     <Compile Include="Screens\Select\Options\BeatmapOptionsRemoveFromUnplayedButton.cs" />
+    <Compile Include="Beatmaps\IO\ArchiveReader.cs" />

```

Comment:
Now that it's not related to beatmaps, it should probably not be in this namespace.